src/mem3.erl

```diff
@@ -254,6 +254,11 @@ allowed_nodes() ->
 nodes_in_zone(Nodes, Zone) ->
     [Node || Node <- Nodes, Zone == mem3:node_info(Node, <<"zone">>)].
 
+non_maintenance_nodes() ->
+    Nodes = [node() | erlang:nodes()],
+    {Modes, _} = rpc:multicall(config, get, ["couchdb", "maintenance_mode"]),
```

Comment:
Here you rely on implementation detail that `rpc:multicall` uses `[node() | erlang:nodes()]`. The order of results is not documented either.